velox/exec/Window.cpp

```diff
@@ -475,6 +475,9 @@ void Window::callApplyLoop(
       break;
     }
   }
+
+  // This tracks the number of rows output in this callApplyLoop.
```

Comment:
This is just a return statement. It is strange to say it tracks something. Maybe this comment belongs to method comment in the .h file.